pkg/lightning/errormanager/errormanager.go

```diff
@@ -89,10 +89,10 @@ const (
 		CREATE TABLE IF NOT EXISTS %s.` + ConflictErrorTableName + ` (
 			task_id     bigint NOT NULL,
 			create_time datetime(6) NOT NULL DEFAULT now(6),
-			table_name  varchar(261) NOT NULL,
```

Comment:
table name should be NOT NULL. For index-type conflict related columns, because lightning may insert record-type confict into this table, they can be NULL